### PR TITLE
test

### PR DESCRIPTION
**Pull request by starfishda**

### CODE REVIEW

File: [CODE] Assignment3/CLASS/DrawingObject.js [hunk 1]
```diff
@@ -218,4 +212,4 @@ function DrawingSetup() {
     }*/
 
    
-}
\ No newline at end of file
+}
```
Comment (banksemi) [inline [CODE] Assignment3/CLASS/DrawingObject.js:215]: The tree seems to have been removed well.
Good 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# Assignment-3
\ No newline at end of file
+# [Graphics] Assignment-3
```
Comment (banksemi) [inline README.md:1]: It is mark to start a review.

File: [CODE] Assignment3/CLASS/DrawingObject.js [hunk 3]
```diff
@@ -202,14 +202,8 @@ function DrawingSetup() {
...
-    
-    /*for (var i = 0; i < 20; i++) {
-        var x = Math.random();
-        var y = Math.random();
-        var object = DrawingObject.Instance(Tree, vec2(x * 1000, 500 + y * 300), vec2(0.15, 0.15));
-        object.z = 5;
-        object.outline = true;
-    }*/
+	
+	// Delete Tree loop
```
Comment (banksemi) [inline [CODE] Assignment3/CLASS/DrawingObject.js:207]: Good.
But, Because Tree is no longer used, I think that it is good to remove class file(Tree.js)

**Review by banksemi — CHANGES_REQUESTED** [review]
Comment

File: [CODE] Assignment3/CLASS/DrawingObject.js (hunk 3 above)
Comment (starfishda) [reply [CODE] Assignment3/CLASS/DrawingObject.js]: Ok!

Comment (starfishda) [reply [CODE] Assignment3/CLASS/DrawingObject.js:207]: I don't think so..

Comment (banksemi) [reply [CODE] Assignment3/CLASS/DrawingObject.js:207]: > I don't think so..

Umm.....
I thought again, the Tree file seems to be used as one of examples.
So I think remaining this file is good.

**Review by banksemi — APPROVED** [review]

**Pull request merged**